### PR TITLE
docs: add docs for cli/values.Options

### PR DESCRIPTION
**Pull request by zalsader**
Add documentation for cli/values.Options

**What this PR does / why we need it**:
Without reading the code, it is hard to know which flag corresponds to which field in `values.Options`. This PR adds the simplest possible documentation.

**If applicable**:
- [x] this PR contains documentation

### CODE REVIEW

**Review by mattfarina — APPROVED** [review]
Due to separation of concerns, I normally wouldn't want a change like this here. This is in a package that's part of the SDK and the flags are part of the client application. But, this file is already commented with the flags. So, it should either be right and up to date or removed. So, approving.

Note, this needs to be approved by a second maintainer before it can be merged.

**Review by technosophos — APPROVED** [review]

**Pull request merged**
